src/components/CodeSnippet/BaseCodeSnippet/index.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import {PrismLight as SyntaxHighlighter} from 'react-syntax-highlighter';
-import {duotoneDark, solarizedlight} from 'react-syntax-highlighter/dist/esm/styles/prism';
-import bashLang from 'react-syntax-highlighter/dist/esm/languages/prism/bash';
-import jsonLang from 'react-syntax-highlighter/dist/esm/languages/prism/json';
-import jsxLang from 'react-syntax-highlighter/dist/esm/languages/prism/jsx';
-import pythonLang from 'react-syntax-highlighter/dist/esm/languages/prism/python';
-import scssLang from 'react-syntax-highlighter/dist/esm/languages/prism/scss';
-import typescriptLang from 'react-syntax-highlighter/dist/esm/languages/prism/typescript';
+import {duotoneDark, solarizedlight} from 'react-syntax-highlighter/dist/cjs/styles/prism';
```

Comment:
just wondering, why did u changed from `esm` to `cjs`?

Reply:
since I added babel config, jest got messed up for node_modules somehow; tests were failing. This was the fix I found.

Reply:
https://github.com/react-syntax-highlighter/react-syntax-highlighter/issues/221